src/__tests__/usePopup.tests.ts

```diff
@@ -22,22 +26,21 @@ describe('usePopup', () => {
         jest.clearAllMocks();
     });
 
-
     test('addSelectedCountry should update selectedCountries', () => {
```

Comment:
Smells of python 👃, we try to use `describes()` for the overall test file and `it()` for individual tests